test/test.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
```

Comment:
We already have `src/tests` directory for tests. Also this has to be fixed for other platforms, openbsd, macos and windows fails.

Reply:
nevermind, I was thinking about something else apparently. Though the file should be renamed to something more meaningful.